.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -47,85 +51,100 @@ body:
         - Linux ARM (Raspberry Pi)
         - Other
     validations:
-      required: false
+      required: true
+
   - type: input
     id: other
     attributes:
-      label: If Other OS, please describe
-      description: Other details
-      placeholder: "Windows, Mac OSX version, Debian, Ubuntu, Arch, etc."
+      label: If other OS or Linux, please describe
+      placeholder: "Details about your OS; if Linux, state your desktop environment (KDE, Gnome, XFCE, etc.)"
     validations:
       required: false
+
+  - type: checkboxes
+    id: checked
+    attributes:
+      label: Are you running DOSBox Staging on a native OS?
+      description: We only support running DOSBox Staging on a native operating system; running on a guest OS in a VM or hypervisor is not supported.
+      options:
+        - label: I am running Staging on a native OS
+          required: true
+
   - type: input
     id: hardware
     attributes:
       label: Relevant hardware info
-      description: Hardware
-      placeholder: "CPU, GPU, device brand/model: e.g. Raspberry Pi3B+, Nvidia, etc"
+      placeholder: "CPU, GPU, device brand/model (e.g., Raspberry Pi3B+, Nvidia RTX 3060, etc.)"
     validations:
       required: false
+
   - type: checkboxes
     id: checked
     attributes:
       label: Have you checked that no other similar issue already exists?
-      description: Searched issues before creating report?
       options:
-        - label: I have searched and not found similar issues.
+        - label: I have searched and have not found similar issues.
           required: true
+
   - type: textarea
     id: description
     attributes:
-      label: A clear and concise description of what the bug is.
-      description: Describe what happens, what software were you running? _Include screenshot if possible_
-      placeholder: "How & When does this occur?"
+      label: A clear and concise description of what the bug is
+      description: Describe what happens, what software were you running? Try to specify the exact version of the game you're running. _Include screenshots if possible._
+      placeholder: "How & when does this occur?"
```

Comment:
Minor: I think removing the '&': "How and when" is better.

Reply:
Good point, changing.